deploy_ephemeral_env.sh

```diff
@@ -16,7 +16,7 @@ source ${CICD_ROOT}/_common_deploy_logic.sh
 # -> use this PR as the template ref when downloading configurations for this component
 # -> use this PR's newly built image in the deployed configurations
 set -x
-export NAMESPACE=$(bonfire namespace reserve --pool ${NAMESPACE_POOL})
```

Comment:
if we want to add this feature, let's try follow the same approach (and syntax) for other global vars , see for example, line 13:

```
# Whether or not to deploy frontends (default: false)
: ${DEPLOY_FRONTENDS:="false"}
```

so something like a comment, a default value, then just use it on line 19

Also would tag @bsquizz  here for review too ^

Reply:
I agree, good suggestion @Victoremepunto -- having all the "options" defined up top is preferred for readability.

Reply:
Yeah sorry I just basically copied the same change I made in Gitlab includes. Now it looks more like the rest.